assemblyline_v4_service/common/base.py

```diff
@@ -23,6 +23,16 @@
 UPDATES_DIR = os.environ.get('UPDATES_DIR', '/updates')
 PRIVILEGED = os.environ.get('PRIVILEGED', 'false') == 'true'
 
+RECOVERABLE_RE_MSG = ["cannot schedule new futures after interpreter shutdown", "can't register atexit after shutdown"]
+
+
+def is_recoverable_runtime_error(error):
+    for msg in RECOVERABLE_RE_MSG:
```

Comment:
nitpick: could be written

if any(msg in str(error) for msg in RECOVERABLE_RE_MSG):
    return True

Reply:
then nvm!